src/routes/About/index.tsx

```diff
@@ -186,6 +186,82 @@ export MODULEPATH=$MODULEPATH:/software/modules/`}</pre>
         Hidden environments can be seen by selecting the 'Show Hidden' option in
         the filter at the top of the environments list.
       </p>
+
+      <h2>Reproducing Environments</h2>
+
+      <p>
+        One of the multitude of benefits of provisioning software this way is the ease
+        of reproducing the environment elsewhere.
+      </p>
+      <p>
+        We currently have two ways of copying and allowing external third-parties to use
+        your SoftPack environments.
+      </p>
+
+      <h3>Upload to DockerHub</h3>
+
+      <p>
+        The first, and simplest, is to upload it to <a href="https://hub.docker.com/">DockerHub</a>
+        so that anyone can clone it as either a Docker image or a Singularity image.
+      </p>
+      <p>
+        NB: This method will only be reproducable on x86_64 machines (that is to say,
+        AMD and Intel based machines); for other architectures, you'll need to use the
+        second method.
+      </p>
+      <p>
+        To upload to DockerHub, first you'll need and account and to generate a
+        <a href="https://app.docker.com/settings/personal-access-tokens">Personal Access Token</a>
+        (Account Settings -&gt; Security -&gt; Personal Access Tokens) that has both Read &amp;
+        Write scopes.
+      </p>
+      <p>
+        Then, on the farm run the following:
+      </p>
+      <pre>{`module load HGI/common/softpack2docker
```

Comment:
I think we've avoided any Sanger-specific stuff in GitHub docs. This level of detail should be in our internal docs.

Reply:
Instead, you should be talking about using the repo behind the softpack2docker module.

Reply:
Hmm, not sure how to do this without mentioning some internal things; even the softpack2docker repo is internal (GitLab). Also, that would not be simple for users to use.

The only thing I can think to do (which I don't like) is having some build-time variable to changes the about text.